src/logic/hooks/useCanTxExecute.tsx

```diff
@@ -5,11 +5,21 @@ type UseCanTxExecuteType = (
   preApprovingOwner?: string,
   txConfirmations?: number,
   existingTxThreshold?: number,
+  txNonce?: string,
```

Comment:
Should we default to `0` here?

Reply:
You wouldn't need to check that it exists before parsing it too then.

Reply:
It could potentially cause issues since the first transaction after safe creation has nonce 0
<img width="408" alt="Screenshot 2022-02-09 at 09 38 16" src="https://user-images.githubusercontent.com/5880855/153157337-a5942b7c-591b-414f-801b-07c1fef00bbc.png">